types/mongodb/index.d.ts

```diff
@@ -1100,25 +1101,30 @@ export interface Collection<TSchema extends { [key: string]: any } = DefaultSche
     estimatedDocumentCount(query: FilterQuery<TSchema>, callback: MongoCallback<number>): void;
     estimatedDocumentCount(query?: FilterQuery<TSchema>, options?: MongoCountPreferences): Promise<number>;
     estimatedDocumentCount(query: FilterQuery<TSchema>, options: MongoCountPreferences, callback: MongoCallback<number>): void;
-    /** http://mongodb.github.io/node-mongodb-native/3.1/api/Collection.html#find */
+    /** http://mongodb.github.io/node-mongodb-native/3.6/api/Collection.html#find */
```

Comment:
Is every change in this PR correct for 3.5 as well? If not, the version in the header needs to be bumped too.

Reply:
MongoDB 4.4 is only _fully_ supported in driver version 3.6. 

Changes in this PR, however, work with driver version 3.5 (and probably even some earlier versions), since the driver just forwards those inputs to the server.